src/internal/admin.py

```diff
@@ -93,7 +93,13 @@ class QuoteAdmin(DefaultAdminWidgetsMixin, UserSearchFieldsMixin, admin.ModelAdm
     autocomplete_fields = ('author',)
 
 
+class LoreAdmin(admin.ModelAdmin):
+    ordering = ('title',)
```

Comment:
I can imagine that it could be useful to also have a column with `slug` 🤔 What do you think?
```suggestion
class LoreAdmin(admin.ModelAdmin):
    list_display = ('title', 'slug')
    ordering = ('title',)
```